keycloak/keycloak
# [KEYCLOAK-12439] Update to Angular 1.7.9

## [KEYCLOAK-12439](https://issues.redhat.com/browse/KEYCLOAK-12439)

Update to Angular 1.7.9

## Verification Steps

1. Remove `themes/src/main/node_modules/` and `themes/src/main/resources/theme/keycloak/common/resources/node_modules/` only to make sure that there's no cache
2. At the root folder of the Keycloak server repository run: 
```
mvn install -Pnpm-update -DskipTests=true && mvn clean install -Pdistribution -DskipTests=true
```
As alternative you can also run:
```
mvn -f testsuite/utils/pom.xml exec:java -Pkeycloak-server -Dkeycloak.profile.feature.account_api=enabled -Dkeycloak.profile.feature.account2=enabled -Dresources
```
4. Check if nothing was broken into the server
5. Run the integration tests as described in https://github.com/keycloak/keycloak/blob/master/testsuite/integration-arquillian/HOW-TO-RUN.md#base-ui-tests.

## Additional Notes
* This PR introduces a ridiculous amount of changes, to be more precise **5,521** changed files. For this reason I made sure to split the commits. So people can focus on [what really matters](https://github.com/keycloak/keycloak/commit/2be4701d51f5ab0cce7a8470e20db44ba43dac9b).
